### ISSUE TITLE
Deprecate RequestFactory

### ISSUE THREAD

**Issue by robhudson** [jbalogh/test-utils#28]
Django 1.4 added `wsgi.input` so this can now point to Django's or be deprecated.

https://github.com/django/django/commit/e2f9c11736db9e23004733760303f605f1cfb924

**Comment by davedash** [comment]
Deprecate... in fact I don't even see the use for most of the things in this repository.  I'd almost say if there is anything useful, just merge it into playdoh

**Comment by robhudson** [comment]
I agree, actually. I'm trying to get zamboni off of test_utils. The Fast Fixture thing is in django-nose which we can use and the few things that are left for us can be added to our base test class -- things like setting the cache timeout to `-1`, etc.

**Comment by jsocol** [comment]
+1 for deprecating this whole package but not necessarily to playdoh. If there are things that are more generic but nose-specific they could go into django-nose

**Comment by davedash** [comment]
I feel like when @robhudson with deprecating (or heck just remove them and bump the version number) there won't be much left.
